.github/workflows/history.yml

```diff
@@ -0,0 +1,50 @@
+name: history
```

Comment:
do you have a sample run of this one too?

Reply:
see other comment with links